src/visualization.ts

```diff
@@ -95,24 +97,40 @@ export class GraphVisualizer {
                             .attr('height', d.relSize * this.config.circleRadiusUnit * 2)
                     )
 
-                    // @ts-ignore
-                    const tip = d3Tip()
-                        .attr('class', 'd3-tip')
-                        .offset([-10, 0])
-                        .html((event: any, d: Node) => {
-                            return d.description;
-                        })
-                    d3.select(this.container).call(tip);
+                    let tip = new Tooltip();
+                    document.body.appendChild(tip.getNode());
```

Comment:
add to body or add to this.container?

Reply:
since the position is absolute, I guess anywhere in the DOM is fine.